build/config/RepoUtilData.json

```diff
@@ -1,26 +1,9 @@
 ﻿{
   "fixedPackages": {
-    "Microsoft.VisualStudio.Language.Intellisense": "14.3.25407",
     "Microsoft.VSSDK.BuildTools": [ "14.3.25420", "15.0.25929-RC2" ],
```

Comment:
Jenkins still uses it in some cases.  Have to do a round of updates before I can remove this.